Microsoft.Toolkit.Uwp.SampleApp/SamplePages/UniformGrid/UniformGridXaml.bind

```diff
@@ -0,0 +1,29 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+  <controls:UniformGrid 
+        FirstColumn="@[FirstColumn:Slider:1:0-3]"
+        Orientation="@[Orientation:Enum:Orientation.Horizontal]"
+        Rows="@[Rows:Slider:0:0-5]"
+        Columns="@[Columns:Slider:0:0-5]">
+    <Border Background="AliceBlue" 
+            Grid.Row="1" Grid.Column="1" 
+            Grid.RowSpan="@[RowSpan:Slider:2:1-3]" 
```

Comment:
Bug with layout: Remove the row and column setters here, put them on the 10th item. run the app and set the Rows and Columns to 1. Only one square should be visible, the element at (0,0). This should be item 1, but instead the 10th item is the only one rendered. It should be in the (1,1) spot and therefore, not rendered.

Reply:
The default behavior of the Grid is to show item 11